ops/terraform/services/pipeline/user-data.sh.tftpl

```diff
@@ -18,12 +18,14 @@ aws ssm get-parameters-by-path \
     --region us-east-1 \
     --query 'Parameters' | jq 'map({(.Name|split("/")|last): .Value})|add' > nonsensitive_pipeline_vars.json
 
+%{ if pipeline_instance == "rda" }
```

Comment:
I see three variables under `/bfd/prod/pipeline/sensitive/ccw` - `ccw_rif_role_external_id`, `ccw_rif_role_principal_arns`, and `s3_bucket_name`. We're sure we don't need these? They do seem to be referenced.

Reply:
I guess the first two are only referenced by other terraform modules but `s3_bucket_name` is references in the app configuration.

Reply:
If not in the ccw hierarchy, it's part of the shared hierarchy at this point. That will continue to be valid configuration until stories/issues subordinate to BFD-2903 are complete.